dev-docs/bidders/pgam.md

```diff
@@ -0,0 +1,101 @@
+---
+layout: bidder
+title: PGAM
+description: Prebid PGAM Bidder Adapter
+biddercode: pgam
+media_types: video,banner
+gdpr_supported: true
+userIds: britepoolId, criteo, id5Id, identityLink, liveIntentId, netId, parrableId, pubCommonId, unifiedId
+schain_supported: true
+coppa_supported: true
+usp_supported: true
+safeframes_ok: true
+prebid_member: true
+pbs: true
```

Comment:
I'll merge this PR for now, but it would be amazing if you could add the following field as well

```
multiformat_supported: will-bid-on-any | will-bid-on-one | will-not-bid
```

This helps sales houses and publishers to avoid issues when using multi format ad units.

More info in #3694